README.md

```diff
@@ -2,6 +2,10 @@
 
 This repository contains documents related to the overall governance of the NUnit organization as well as the individual projects under NUnit. It is currently under development.
 
+## Code of Conduct
+
+Please note that all NUnit projecst are released with a [Contributor Code of Conduct](CODE_OF_CONDUCT.md). By participating in these projects you agree to abide by its terms.
```

Comment:
I didn't know they recommended that wording. In that case I have no opinion either way besides a small typo fix.